build-kernels-clang.sh

```diff
@@ -7,7 +7,10 @@ CLANG_A11=$ANDROID_ROOT/prebuilts/clang/host/linux-x86/clang-r353983c/bin/
 CLANG_A12=$ANDROID_ROOT/prebuilts/clang/host/linux-x86/clang-r416183b/bin/
 CLANG_A13=$ANDROID_ROOT/prebuilts/clang/host/linux-x86/clang-r450784d/bin/
 
-if  [ -d "$CLANG_A13" ]; then
+if [ "$use_system_clang" = true ]; then
```

Comment:
I was thinking, since I added back a `build-kernels-gcc.sh` (using host GCC instead), can't we simplify this by just having a `build-kernels-host-clang.sh` and `build-kernels-aosp-clang.sh` respectively? There's barely anything that the scripts share, except that `aosp` could call the `host` file because it's the same minus setting an alternate `PATH`.

Reply:
please use build-kernels-host-clang.sh and build-kernels-aosp-clang.sh

Reply:
And `build-kernels-aosp-clang.sh` can, after adding `CLANG` to `PATH`, simply `source` `build-kernels-host-clang.sh` (or just `build-kernels-clang.sh`) which then contains:


```sh
# Build command
BUILD_ARGS="LLVM=1 LLVM_IAS=1 CC=clang"
PATH=$CLANG:$PATH
# source shared parts
. "${0%/*}/build_shared.sh"
```